web/src/main/java/com/bazaarvoice/emodb/web/migrator/LocalMigratorMonitor.java

```diff
@@ -365,27 +359,4 @@ private void initializeAllActiveMigrations() {
             _log.warn("Failed to initialize active migration count", e);
         }
     }
-
-    private void scheduleOverrunCheck(ScanStatus status) {
-        final String migratorId = status.getScanId();
-
-        Instant now = Instant.now();
-        Instant overrunTime = status.getStartTime().toInstant().plus(OVERRUN_MIGRATION_TIME);
```

Comment:
Not related to this line, but static final value of OVERRUN_MIGRATION_TIME can be removed.

Reply:
whoops, I could've sworn i pulled it out. It must've returned during my rebase.